### PR TITLE
Ticker Tape Configuration Updates.  

### PR DESCRIPTION
**Pull request by mhotan**
Validation done in Ticker Tape Configuration class.  No more relying on hardcoded default attributes.  Its  in the configuration file or overridden in the command line.

### CODE REVIEW

File: src/main/scala/com/socrata/tickertape/TickerTape.scala [hunk 1]
```diff
@@ -1,43 +1,25 @@
...
-  logger info s"${this.getClass.getSimpleName} configured to write metrics to $dataDir"
-  logger info s"${this.getClass.getSimpleName} configured to write $batchSize metrics every $sleepTime milliseconds"
-  logger info s"Starting ${this.getClass.getSimpleName}..."
-
-  val idPart = new MetricIdPart("metrics-internal-test")
-  while (true) {
-    logger debug s"Writing $batchSize metrics"
-    0 until batchSize foreach { i =>
-      queue.create(idPart, Fluff(s"fake-metric-$i"), 1)
+  override def run(): Unit = 0 until config.batchSize foreach { i =>
```
Comment (coyotebush) [inline src/main/scala/com/socrata/tickertape/TickerTape.scala]: Two nested loops doesn't look right.

File: src/main/scala/com/socrata/tickertape/TickerTape.scala [hunk 2]
```diff
@@ -1,43 +1,25 @@
...
+    logger debug s"Writing ${config.batchSize} metrics"
+    0 until config.batchSize foreach { i =>
+      queue.create(config.metricsEntityId, Fluff(s"fake-metric-$i"), 1)
     }
-    logger debug s"Sleeping $sleepTime milliseconds"
-    Thread.sleep(sleepTime)
+    logger debug s"Sleeping ${config.sleepTime} milliseconds"
```
Comment (coyotebush) [inline src/main/scala/com/socrata/tickertape/TickerTape.scala]: I guess this statement is still nearly accurate, even though the scheduling is done elsewhere. (And a Chronos-managed service wouldn't need any of this, right?)

File: src/main/scala/com/socrata/tickertape/TickerTape.scala (hunk 1 above)
Comment (mhotan) [inline src/main/scala/com/socrata/tickertape/TickerTape.scala]: Haha Whoops

**Comment by coyotebush** [comment]
:shipit:

**Pull request merged**